test/release/examples/benchmarks/shootout/chameneosredux.chpl

```diff
@@ -74,6 +74,12 @@ proc simulate(numChameneos) {
   // print information about the population once we're done
   //
   printInfo(chameneos);
+
+  //
+  // hasta las chameneos, baby.
+  //
+  destroyChameneos(chameneos);
+  delete meetingPlace;
```

Comment:
Sounds good.  Thanks.
THH
________________________________
From: Lydia Duncan [notifications@github.com]
Sent: Wednesday, June 17, 2015 1:05 PM
To: chapel-lang/chapel
Cc: Tom Hildebrandt
Subject: Re: [chapel] String as rec June 11 leak fixes. (#2067)


In test/release/examples/benchmarks/shootout/chameneosredux.chpl<https://github.com/chapel-lang/chapel/pull/2067#discussion_r32668613>:

> @@ -74,6 +74,12 @@ proc simulate(numChameneos) {
>    // print information about the population once we're done
>    //
>    printInfo(chameneos);
> +
> +  //
> +  // hasta las chameneos, baby.
> +  //
> +  destroyChameneos(chameneos);
> +  delete meetingPlace;


I can make the chameneos change. If the iterator change doesn't make a difference outside of your branch, we might as well leave it there :)

—
Reply to this email directly or view it on GitHub<https://github.com/chapel-lang/chapel/pull/2067/files#r32668613>.